src/ConfigureMentions.php

```diff
@@ -99,7 +99,7 @@ public static function addPostId($tag)
     {
         $post = CommentPost::find($tag->getAttribute('id'));
 
-        if ($post) {
+        if ($post->user) {
```

Comment:
Hmm, this breaks the check for `$post` (the referenced post could be deleted, too). So, at the very least this should be...
```suggestion
        if ($post && $post->user) {
```

However, I am unsure whether we want to wrap the whole block in this execution or just the line setting the `displayname` attribute... in which case, we'd need to consider how a missing attribute would be handled... or which fallback value we'd add instead here.

Reply:
@franzliedke Oh no, apologies for the oversight 

I took another look, and did some more thorough testing. I tested adding the suggested change, as well as adding some code like below, to add a display_name of '[deleted'] if the $post->user attribute was missing.

`            $tag->setAttribute('displayname',
              isset($post->user) ? $post->user->display_name : '[deleted]'
            );
`

I tested both methods and they both apparently worked ok, but I found a different error using the method above (display_name = [deleted]) when restoring a deleted post which mentioned a deleted user. I have attached the log below (it won't let me attach it here for some reason)

I will commit your suggestion to the PR.